app/src/main/java/com/conupods/OutdoorMaps/Models/PointsOfInterest/Place.java

```diff
@@ -65,19 +65,19 @@ public void setId (String id)
         this.id = id;
     }
 
-    public String getPlace_id ()
+    public String getPlaceId()
     {
-        return place_id;
+        return placeId;
     }
 
-    public void setPlace_id (String place_id)
+    public void setPlaceId(String placeId)
     {
-        this.place_id = place_id;
+        this.placeId = placeId;
     }
 
     @Override
     public String toString()
     {
-        return "ClassPojo [types = "+", icon = "+", rating = "+", photos = "+photos+", reference = "+", user_ratings_total = "+", scope = "+", name = "+name+", opening_hours = "+", geometry = "+", vicinity = "+", id = "+id+", plus_code = "+", place_id = "+place_id+"]";
+        return "ClassPojo [types = "+", icon = "+", rating = "+", photos = "+photos+", reference = "+", user_ratings_total = "+", scope = "+", name = "+name+", opening_hours = "+", geometry = "+", vicinity = "+", id = "+id+", plus_code = "+", place_id = "+ placeId +"]";
```

Comment:
Do not change the name of this attribute, it must match the attribute name that is within the JSON response object. Revert change.